setup.py

```diff
@@ -955,11 +987,67 @@ def build_torch_extension_v2(build_ext, options, torch_version):
     build_ext.build_extension(torch_mpi_lib_v2)
 
 
+def build_cmake(build_ext, ext, output_dir, options):
+    # CMake should come with pip requirement 'cmake', but let's verify
+    cmake_bin = 'cmake'
+    if not have_cmake:
+        import cmake
+        cmake_bin = os.path.join(cmake.CMAKE_BIN_DIR, 'cmake')
+        if os.access(cmake_bin, os.X_OK):
+            os.chomod(cmake_bin, stat.S_IXUSR)
```

Comment:
We may need more than user executable (at least group, possibly other).

Reply:
Thanks! Added mode to include group and others.